src/components/utils/popover/components/popover-item/popover-item-default/popover-item-default.ts

```diff
@@ -86,11 +93,14 @@ export class PopoverItemDefault extends PopoverItem {
    * Constructs popover item instance
    *
    * @param params - popover item construction params
+   * @param renderParams - popover item render params.
+   * The parameters that are not set by user via popover api but rather depend on technical implementation
    */
-  constructor(params: PopoverItemDefaultParams) {
+  constructor(params: PopoverItemDefaultParams, renderParams?: PopoverItemRenderParams) {
     super();
 
     this.params = params;
+    this.hintPosition = renderParams?.hint?.position || 'right';
```

Comment:
seems redundant. Why not just use `params.hint.position`?

Reply:
I do not want to let users set the position manually

Reply:
so, seem like we need to split parameters that can be altered by users and the ones that are 'technical'